IDEAS/Fluid/Taps/BaseClasses/PartialTap.mo

```diff
@@ -0,0 +1,240 @@
+          lineColor={28,108,200},
+          fillColor={28,108,200},
+          fillPattern=FillPattern.Solid,
+          lineThickness=1),
+        Line(
+          points={{-90,0},{-80,0}},
+          color={238,46,47},
+          thickness=1),
+        Line(
+          points={{80,0},{90,0}},
+          color={28,108,200},
+          thickness=1),
+        Polygon(
+          points={{-20,40},{20,40},{12,80},{-12,80},{-20,40}},
+          lineColor={0,0,0},
+          fillColor={240,240,240},
+          fillPattern=FillPattern.Solid,
+          lineThickness=1),
+        Line(points={{-8,80},{-14,40}}, color={162,162,162}),
+        Line(points={{0,80},{0,78},{0,40}}, color={162,162,162}),
+        Line(points={{8,80},{14,40}}, color={162,162,162}),
+        Line(points={{-4,80},{-8,40}}, color={162,162,162}),
+        Line(points={{4,80},{8,40}}, color={162,162,162}),
+        Polygon(
+          points={{-70,68},{-70,48},{-76,42},{-76,32},{-70,26},{-62,26},{-56,32},
+              {-56,42},{-62,48},{-62,68},{-70,68}},
+          fillColor={238,46,47},
+          fillPattern=FillPattern.Solid,
+          pattern=LinePattern.None),
+        Polygon(
+          points={{-70,86},{-70,48},{-76,42},{-76,32},{-70,26},{-62,26},{-56,32},
+              {-56,42},{-62,48},{-62,86},{-66,90},{-70,86}},
+          lineColor={0,0,0},
+          lineThickness=0.5),
+        Line(
+          points={{-70,76},{-66,76}},
+          color={0,0,0},
+          thickness=0.5),
+        Line(
+          points={{-66,72},{-70,72}},
+          color={0,0,0},
+          thickness=0.5),
+        Line(
+          points={{-66,68},{-70,68}},
+          color={0,0,0},
+          thickness=0.5),
+        Line(
+          points={{-70,64},{-66,64}},
+          color={0,0,0},
+          thickness=0.5),
+        Line(
+          points={{-70,60},{-66,60}},
+          color={0,0,0},
+          thickness=0.5),
+        Line(
+          points={{-70,56},{-66,56}},
+          color={0,0,0},
+          thickness=0.5)}),
+    Documentation(info="<html>
+<p><b>Description</b></p>
+<p>Partial model of a domestic hot water (DHW) tap. The tap is modelled as a thermostatic mixing valve.</p>
+<p>The model has two flowPorts and a realInput: </p>
+<ul>
+<li><i>port_hot</i>: connection to the hot water source (designation: <i>hot</i> )</li>
+<li><i>port_cold</i>: connection to the inlet of cold water in the hot water source (designation: <i>cold </i>)</li>
+<li><i>m_flow_Set</i>: desired flowrate of DHW water, equivalent at a user defined set point temperature</li>
+</ul>
+<p>The model tries to reach the given DHW flow rate at a the desired mixing temperature <i>TSet </i>by mixing the hot water with cold water. The resulting hot flowrate (<i>m_flow_Hot </i>) will be extracted automatically from the hot source (via <i>port_hot </i>). This same flow rate will be injected at <i>TCold</i> in the production system through the connection of <i>port_cold</i> to the hot source. </p>
```

Comment:
This is not easily readable. In the Buildings library development info, they mention: "Keep the line length to no more than around 80 characters."

Reply:
ok!